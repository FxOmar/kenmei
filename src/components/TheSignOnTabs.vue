<template lang="pug">
el-tabs(v-model="activeTab" stretch)
  el-tab-pane(label="Sign In" name="signIn")
    the-sign-in(@resetPassword="$emit('componentChanged', 'TheResetPassword')")
  el-tab-pane(label="Sign Up" name="signUp")
    the-sign-up
</template>

<script>
  import { Tabs, TabPane } from 'element-ui';

  import TheSignIn from '@/components/TheSignIn';
  import TheSignUp from '@/components/TheSignUp';

  export default {
    components: {
      TheSignIn,
      TheSignUp,
      'el-tabs': Tabs,
      'el-tab-pane': TabPane,
    },
    data() {
      return {
        activeTab: 'signIn',
      };
    },
  };
</script>
