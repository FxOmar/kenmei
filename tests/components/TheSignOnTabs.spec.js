import { shallowMount } from '@vue/test-utils';
import TheSignOnTabs from '@/components/TheSignOnTabs.vue';

describe('TheSignOnTabs.vue', () => {
  it('renders progress bar', () => {
    const signOnTabs = shallowMount(TheSignOnTabs);

    expect(signOnTabs.html()).toMatchSnapshot();
  });
});
