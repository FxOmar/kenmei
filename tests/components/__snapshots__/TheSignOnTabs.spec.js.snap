// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSignOnTabs.vue renders progress bar 1`] = `
<el-tabs-stub value="signIn" tabposition="top" stretch="true">
  <el-tab-pane-stub label="Sign In" name="signIn">
    <the-sign-in-stub></the-sign-in-stub>
  </el-tab-pane-stub>
  <el-tab-pane-stub label="Sign Up" name="signUp">
    <the-sign-up-stub></the-sign-up-stub>
  </el-tab-pane-stub>
</el-tabs-stub>
`;
